brainframe/api/stubs/capsules.py

```diff
@@ -28,6 +29,38 @@ def get_capsules(self, timeout=DEFAULT_TIMEOUT) -> List[Capsule]:
         capsules, _ = self._get_json(req, timeout)
         return [Capsule.from_dict(d) for d in capsules]
 
+    def set_capsule(self, storage_id: int,
+                    source_path: Optional[Path] = None,
+                    timeout=DEFAULT_TIMEOUT) -> Capsule:
+        """Sends capsule data to the server to be loaded and initialized.
+
+        :param storage_id: The ID of the raw capsule data
+        :param source_path: The path to the source code on the development
+            machine, if available. Providing this allows stack traces to point
+            to the correct source location.
+        :param timeout: The timeout to use for this request
+        :return: The loaded capsule
+        """
+        req = f"/api/plugins"
+        req_object = {
+            "storage_id": storage_id,
+            "dev_options": {
+                "source_path": str(source_path),
+            },
+        }
+        capsule = self._put_json(req, timeout, json.dumps(req_object))
+        return Capsule.from_dict(capsule)
+
+    def delete_capsule(self, capsule_name: str, timeout=DEFAULT_TIMEOUT):
+        """Unloads a capsule. This method may only be used with capsules that
+        were loaded through the REST API.
+
+        :param capsule_name: The name of the capsule to unload
```

Comment:
Method is called "delete" but param describes it as "unload"
We should unify our terminology here